src/pydata_sphinx_theme/assets/styles/components/_breadcrumbs.scss

```diff
@@ -5,14 +5,25 @@ ul.bd-breadcrumbs {
   list-style: none;
   padding-left: 0;
   display: flex;
-  flex-wrap: wrap;
+  flex-wrap: nowrap;
 
   // Font size slightly smaller to avoid cluttering up space too much
   font-size: 0.8rem;
 
   li.breadcrumb-item {
     display: flex;
     align-items: center;
```

Comment:
align-items was changed to center and was this conflicting, I resolved the conficlt.